addons/point_of_sale/tests/test_pos_basic_config.py

```diff
@@ -889,6 +889,27 @@ def open_and_check(pos_data):
         open_and_check(pos01_data)
         open_and_check(pos02_data)
 
+    def test_pos_orders_sequencing(self):
+        """ This test check if the sequence in correctly set """
+
+        sequence = self.env['ir.sequence'].search([('code', '=', 'pos.session')])
+        sequence.prefix = '/'
+        sequence.write({'number_next_actual': 1000})
+        name = self.config.name
+
+        self.open_new_session(0)
+        self.assertEqual(self.pos_session.name, name + '/01000')
+
+        self.pos_session.close_session_from_ui()
+
+        sequence.prefix = 'TEST/'
+        sequence.write({'number_next_actual': 1001})
+
+        self.open_new_session(0)
+        self.assertEqual(self.pos_session.name, 'TEST/01001')
+
+        self.pos_session.close_session_from_ui()
```

Comment:
It's not required to close the session, so let's not do it as it will also reduce the test duration.